src/graph/unit_graph.cc

```diff
@@ -851,24 +974,41 @@ HeteroSubgraph UnitGraph::EdgeSubgraph(
 }
 
 HeteroGraphPtr UnitGraph::CreateFromCOO(
-    int64_t num_vtypes, int64_t num_src, int64_t num_dst, IdArray row, IdArray col) {
+    int64_t num_vtypes, int64_t num_src, int64_t num_dst, IdArray row, IdArray col,
+    SparseFormat restrict_format) {
```

Comment:
Can we have a default `Any` value for `restrict_format` so it's easier to use it.

Reply:
They do have a default value in `unit_graph.h`.